bridge/opencensus/internal/oc2otel/span_context_test.go

```diff
@@ -13,6 +13,14 @@ import (
 )
 
 func TestSpanContextConversion(t *testing.T) {
+	tsOc, _ := tracestate.New(nil,
```

Comment:
I propose similar changes as I proposed for `TestSpanContextConversion`

Reply:
I think this still needs work.

Reply:
Could you be more specific? I have committed your changes. But the change didn't compile, so spend some time fixed it.

Reply:
> I have committed your changes. But the change didn't compile, so spend some time fixed it.

Thanks for fixing that. GitHub won't let me suggest something that has not been touched in the PR, so I cannot also suggest the import path, which leads to compile issue.

> Could you be more specific?

I mean we could also add more checking for this `TestSpanContextConversion`, similar like 

```go
			// Ensure the otel tracestate and oc tracestate has the same header output
			_, ts := httpFormatOc.SpanContextToHeaders(tc.expected)
			assert.Equal(t, tc.expectedTracestate, ts)
			assert.Equal(t, tc.expectedTracestate, tc.input.TraceState().String())

			// The reverse conversion should yield the original input
			input := oc2otel.SpanContext(output)
			assert.Equal(t, tc.input, input)
```

This ensures the conversion won't change the header output from both otel and oc.

Reply:
Got it and thanks for the code snippet.  Didn't know Github has that restriction. I have made the changes accordingly. Let me know if there's anything else to change.